File: CarFleet-Project/Models/Tables/VehicleImage.cs
namespace CarFleet_Project.Models.Tables
{
    public class VehicleImage
    {
        public int imageId { get; set; }
        public string imageType { get; set; } = "";
        public string imageName { get; set; } = "";
        public int vehicleId { get; set; }
        public virtual Vehicle vehicle { get; set; } = null!;
        public byte[] image { get; set; } = null!;


    }
}

File: CarFleet-Project/ClientApp/src/models/VehicleImage.ts
import { SafeUrl } from '@angular/platform-browser';

export class VehicleImage {
  imageId: number;
  imageType: string;
  imageName: string;
  vehicleId: number;
  imageBase64: string;
  imageUrl: SafeUrl;

  constructor() {
    this.imageId = 0;
    this.imageType = "";
    this.imageName = "";
    this.vehicleId = 0;
    this.imageBase64 = "";
    this.imageUrl = "";
  }
}
